aspnetcore-webapi/RecordsAsModelClasses/RecordsAsModelClasses/Core/Context/CarDbContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RecordsAsModelClasses.Core.Entities;
+
+namespace RecordsAsModelClasses.Core.Context;
+
+public class CarDbContext : DbContext
+{
+    public CarDbContext(DbContextOptions<CarDbContext> options)
+        : base(options)
+    {
+    }
+
+    public DbSet<Car> Cars { get; set; }
+
+    public DbSet<VintageCar> VintageCars { get; set; }
```

Comment:
We should swap order between constructor and properties. 
Properties and fields should always be first.

Reply:
This is done